content/log/processing.md

```diff
@@ -0,0 +1,199 @@
+---
+title: Processing
+kind: Documentation
+autotocdepth: 2
+hideguides: true
+customnav: lognav
+---
+
+<div class="alert alert-info">
+Datadog's log management solution is is currently in private beta. If you would like to apply to it, please fill out <a href="https://www.datadoghq.com/log-management/">this form</a>.
+</div>
+
+## Overview
+
+To access the processing panel please use the upper left menu:
+{{< img src="log/processing/processing_panel.png" alt="Pipelines pannel" >}}
+
+Your processing view should look like this:
+{{< img src="log/processing/processing_view.png" alt="Pipelines view" >}}
+
+## Processing Pipelines 
+### Pipelines Goal 
+
+**A processing pipeline applies over a filtered subset of incoming logs a list of sequential processors.**
+
+With pipelines you have the opportunity to parse & enrich your logs through a sequential chains of [processors](#processors) in order to extract meaningful information or to extract attributes from semi-structured text to re-use them as [facet](/log/explore/#facets).
```

Comment:
> With pipelines you have the opportunity to parse & enrich your logs through a sequential chains of [processors]

"With pipelines, you can parse and enrich your logs by chaining them sequentially through [processors]. This lets you extract meaningful information or attributes from semi-structured text to reuse them as [facets]."

Though, can we also omit "semi-structured text"? Doesn't this just refer to, well, the logs? Or instead of omit, change it to "log text", unless the fact that  it expects "semi-structured" logs is an important distinction to make.

Reply:
👍